auth/client/src/main/java/org/wildfly/security/auth/client/ElytronMessages.java

```diff
@@ -65,6 +65,15 @@ ConfigXMLParseException xmlInvalidPortNumber(@Param XMLStreamReader reader, Stri
     @Message(id = 1029, value = "Invalid host specification \"%s\"")
     IllegalArgumentException invalidHostSpec(String hostSpec);
 
+    @Message(id = 1030, value = "Invalid iteration count, must not be 0")
```

Comment:
Maybe reword this to indicate that the minimum value is 1

Reply:
I removed these exceptions